proposals/type-classes.md

```diff
@@ -171,10 +171,10 @@ transform(listOf(1), { it + 1 }) // does not compile: No `Functor<List>` instanc
 
 ## Language changes
 
-- Add `with` to require instances evidences in both function and class/object declarations
-- Add `extension` to provide instance evidences for a given type class
+* Add `with` to require evidence of type class instances in both function and class/object declarations.
+* Add `extension` to provide instance evidence for a given type class.
```

Comment:
It reads better to treat "evidence" as an uncountable noun.

Reply:
Also adding full stops to the end of the items as this is done for all other lists currently.